### ISSUE TITLE
Feature Request (simple): increase the hardness value of all concrete by just a tiny bit

### ISSUE THREAD

**Issue by OmegaCenti** [Haggle1996/RevolutionPack#507]
InTitle

Please :D

**Comment by Aivech** [comment]
Why?

**Comment by OmegaCenti** [comment]
its a wee bit too easy to just bust through your concrete floor, .
Scenario, your inventory is full (except for the thing you are breaking is
already stacked in your inventory.) you break the thing, your pick is held
down for 1 ms too long, and now you have smashed up a line of concrete in
the direction you were looking. You get to either, put something away, a
mild convenience, or you get to throw something equally important out of
your inventory, moved the concrete to the hotbar, and replace. Its an
elimination of a possible minor inconvenience. So Damachinator, why not?

**Comment by jusvit** [comment]
I think this is a mod thing, i don't think haggle can change the hardness directly himself, but yeah, i think it would be better if concrete ( which in real life is pretty strong) should be at least a little more harder than dirt...

**Comment by Aivech** [comment]
Rotarycraft concrete is pretty hard.

**Comment by Eb-** [comment]
I'd like to see IE concrete harder myself, way too easy to break the stuff accidentally.

**Comment by DaMachinator** [comment]
Unfortunately, unless IE has a config for it, I don't know of any ways to change block hardness.

**Comment by aurumvorax** [comment]
I would prefer this as well, IE concrete is pretty, and nice to build with, but it's a little too easy to be breaking down the arc furnace to move it and just plough through the wall and floor, letting in the zombie hordes, who now have SMG's.  Not that I have any specific examples in mind or anything....

**Comment by Aivech** [comment]
Do we know if there is any way for Haggle to actually do this?

**Comment by Barhandar** [comment]
There is.

**Comment by Haggle1996** [comment]
Yes, it's doable. I haven't done it because I kept thinking I should go through all the materials and make them make more sense. Making mountains out of mole hills
